src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -0,0 +1,187 @@
+    If there are no results, it's just the timestamp
+    {'time': 1719012000}, {'time': 1719018000}, {'time': 1719024000}
+
+    If there are results, the count is added
+    {'time': 1721300400, 'count': 2}
+    """
+    formatted_data = []
+    for datum in data.data.get("data", []):
+        ts_point = TimeSeriesPoint(timestamp=datum.get("time"), value=datum.get("count", 0))
+        formatted_data.append(ts_point)
+    return formatted_data
+
+
+def translate_direction(direction: int) -> str:
+    """
+    Temporary translation map to Seer's expected values
+    """
+    direction_map = {
+        AlertRuleThresholdType.ABOVE: "up",
+        AlertRuleThresholdType.BELOW: "down",
+        AlertRuleThresholdType.ABOVE_AND_BELOW: "both",
+    }
+    return direction_map[AlertRuleThresholdType(direction)]
+
+
+def get_project_id_from_rule(rule: AlertRule) -> int | None:
+    project = None
+    subscriptions = rule.snuba_query.subscriptions.all()
+    if subscriptions:
+        project = subscriptions[0].project
+        if not project:
+            return None
+
+    if project:
+        return project.id
+
+    return None
+
+
+def send_historical_data_to_seer(rule: AlertRule, user: User) -> BaseHTTPResponse:
+    base_error_response = BaseHTTPResponse(
+        status=status.HTTP_400_BAD_REQUEST,
+        reason="Something went wrong!",
+        version=0,
+        version_string="HTTP/?",
+        decode_content=True,
+        request_url=SEER_ANOMALY_DETECTION_STORE_DATA_URL,
+    )
+    if not features.has("organizations:anomaly-detection-alerts", rule.organization, actor=user):
+        base_error_response.reason = "You do not have the anomaly detection alerts feature enabled."
+        return base_error_response
+
+    project_id = get_project_id_from_rule(rule)
+    if not project_id:
+        logger.error(
+            "No project associated with rule. Skipping sending historical data to Seer",
+            extra={
+                "rule_id": rule.id,
+                "project_id": project_id,
+            },
+        )
+        base_error_response.reason = "No project associated with rule. Cannot create rule."
+        return base_error_response
+
+    snuba_query = SnubaQuery.objects.get(id=rule.snuba_query_id)
+    time_period = int(snuba_query.time_window / 60)
+    historical_data = fetch_historical_data(rule, snuba_query)
+
+    if not historical_data:
+        base_error_response.reason = "No historical data available. Cannot create rule."
+        return base_error_response
+
+    formatted_data = format_historical_data(historical_data)
+
+    if not rule.sensitivity or not rule.seasonality or rule.threshold_type is None:
+        # this won't happen because we've already gone through the serializer, but mypy insists
+        base_error_response.reason = (
+            "Cannot create rule - missing expected configuration for a dynamic alert."
+        )
+        return base_error_response
+
+    ad_config = ADConfig(
```

Comment:
Lets spell out full words rather than abbreviate

Reply:
this was named by the ai/ml team - I can rename the variable but the typeddict should match as closely to the seer data as possible